docs/spec_draft.md

```diff
@@ -1406,6 +1407,76 @@ and produces a `result` tensor. More formally,
 ```
 
 [Back to Ops](#index-of-ops)
+
+## stablehlo.rng
+
+### Semantics
+
+Constructs an output of a given shape with random numbers generated given
```

Comment:
Constructs a `result` tensor of a given shape `shape` with random numbers generated using `rng_distribution` algorithm.  If `rng_distribution` $=$ `UNIFORM`, then the random numbers are generated following the uniform distribution over the interval  `[a,b)`. If `rng_distribution` $=$ `NORMAL`, then the random numbers are generated following the  normal distribution with mean = `a` and standard deviation = `b`.